src/main/asciidoc/_chapters/protobuf.adoc

```diff
@@ -148,3 +148,67 @@ consider extending it also in
 Going forward, we will provide a new module of common types for use
 by CPEPs that will have the same guarantees against change as does our
 public API. TODO.
+
+=== protobuf changes for hbase-3.0.0 (HBASE-23797)
+Since hadoop also shades protobuf and bumps the version to 3.x, there is no
+reason for us to stay on protobuf 2.5.0 any more.
+
+In 3.0.0, the hbase-protocol module has been purged, the CPEP implementation
```

Comment:
Say hbase-3.0.0 because 3.0.0 could get confused w/ hadoop 3.0.0 which you just mentioned